### ISSUE TITLE
Loss increasing on every epoch

### ISSUE THREAD

**Issue by hiba007** [experiencor/keras-yolo2#248]
I am training on yolov2 architecture with VOC dataset but the loss is increasing on every epoch
I downloaded the train/val dataset and split the dataset with 0.7 ratio
